### ISSUE TITLE
"mask_nonlinear" choose problem 

### ISSUE THREAD

**Issue by xiaoyiming** [kaituoxu/Conv-TasNet#23]
Hi kaituo xu:
      when I train  your code on a new datasets I meet some problem.  when I choose  "mask_nonlinear"  of relu , I can get similar results with you. However, when I choose ""mask_nonlinear"  of softmax,  the training loss is always -1. 4 .  Did you meet the same problem ,when you choose  "mask_nonlinear"  of softmax?   the choose in the Conv-tasnet is softmax, why do you choose relu?
